### ISSUE TITLE
[BITV] 9.4.1.2/3.9 - Although the dialog "Choose target folder" has been marked as such with the appropriate role role="dialog", the implementation is lacking an aria-modal="true" attribute. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37106]
- [x] https://github.com/nextcloud/server/pull/39382

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n9

</details>

- [x] 9.4.1.2/10.9 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n9
